packages/web-react/src/components/Grid/README.md

```diff
@@ -0,0 +1,28 @@
+# Grid
+
+Use Grid to build multiple column layouts. This Grid works on twelve column system, and it contains variants of 12, 6, 4, 3, 2, and 1 column for each breakpoint. And additionally one centered narrow column.
+
+```jsx
+<Grid cols={6} desktop={4} tablet={3} mobile={2} narrow>
```

Comment:
A more realistic example would do a better job I believe 🙂. Since `mobile` is redundant (see `Grid` types) and—if you agree—`narrow` is going to become something like `layout="narrow"`, and it's mutually exclusive with other props, I'd offer two examples that present the `Grid` component in a more real-world fashion:

```jsx
<Grid cols={2} tablet={3} desktop={4}>
  <span>col 1</span>
  <span>col 2</span>
  <span>col 3</span>
  <span>col 4</span>
  <span>col 5</span>
  <span>col 6</span>
</Grid>
```

… and the layout prop:


```jsx
<Grid layout="narrow">
  <div>content</div>
</Grid>
```